packages/@tailwindcss-upgrade/src/codemods/template/migrate-legacy-classes.ts

```diff
@@ -72,11 +71,6 @@ export async function migrateLegacyClasses(
   designSystem: DesignSystem,
   _userConfig: Config | null,
   rawCandidate: string,
-  location?: {
-    contents: string
-    start: number
-    end: number
-  },
```

Comment:
The main `migrate` function will still receive the location, but each individual migration will not